### ISSUE TITLE
Copycat/clone of LibreTube on Google Play

### ISSUE THREAD

**Issue by Poussinou** [libre-tube/LibreTube#3301]
### Steps to reproduce

Just found the apps below on Google Play :
* https://play.google.com/store/apps/details?id=tubevideo.playtube.mp3
* https://play.google.com/store/apps/details?id=melotube.playtube.mp3

Both apps are uploaded by this developer https://play.google.com/store/apps/dev?id=6130128057652620670

He also uploaded copycats of Newpipe as you can see [here](https://github.com/TeamNewPipe/NewPipe/issues/539#issuecomment-1464932686). I expect ads to be added shortly in all this clones...

### Expected behavior

This copycats don't respect GNU GPL v3 as far as I can see.

Expected behavior: Send a DMCA request to Google Play. It violates their rule anyway, should be easy to take down...

### Actual behavior

Still online as of 13/03/2023

### LibreTube version

0.12.1

### Android version

Android 10

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Thanks for creating the issue!

It seems like a Google account is being required in order to report a copycat to Google, so I'd be happy if someone could report the apps (I neither have nor want to create an account). It seems like https://github.com/TeamNewPipe/NewPipe/issues/539#issuecomment-1399222536 could be the best way to proceed as it gives some general advice on how copycats were already taken down successfully.

**Comment by Ahmedinejadi** [comment]
Let people make money.  They are not defrauding anyone, they have the right to modify and use open source code.  They don't harm anyone.  Don't you have any other job?  Are you quitting your job and developing complaining techniques?  Why are you having such a hard time?

**Comment by ARBoyGo** [comment]
@Ahmedinejadi Looks like you are one of the clone, Brother, Making money from someone's work is not best, LibreTube is Non Profit, And yes anyome can modify and use the code as they want but shouldn't go against our policy, You can fork it, No problem, But making profit out of it, We are against it. 

May Allah bless you brother.

**Comment by Poussinou** [comment]
> Let people make money. They are not defrauding anyone, they have the right to modify and use open source code. They don't harm anyone. Don't you have any other job? Are you quitting your job and developing complaining techniques? Why are you having such a hard time?

Lol that guy created his account just to comment this...

**Comment by KoremuMekuro** [comment]
> Let people make money. They are not defrauding anyone, they have the right to modify and use open source code. They don't harm anyone. Don't you have any other job? Are you quitting your job and developing complaining techniques? Why are you having such a hard time?

This project is under GPL-3.So where are the code of this copycats.If you don't understand,gpl-3 requiers you to release the source code of your project of your project uses code from other project that under gpl-3(this means that at least that "dev" must release source code of his copycats).

**Comment by ARBoyGo** [comment]
> Lol that guy created his account just to comment this...

Yes that's right, Lol, I didn't checked his profile, Seriously. I'm right now Why did I say blessings on god's name 😂.

**Comment by heartlog** [comment]
> > Let people make money. They are not defrauding anyone, they have the right to modify and use open source code. They don't harm anyone. Don't you have any other job? Are you quitting your job and developing complaining techniques? Why are you having such a hard time?
> 
> Lol that guy created his account just to comment this...

Well, I guess he is the one of copycat that uploded modified ver of LibreTube on playstore.to make some money offcourse..

**Comment by matkoniecz** [comment]
> It seems like a Google account is being required in order to report a copycat to Google, so I'd be happy if someone could report the apps

I guess that the first required step would be to (1) install app (2) request app source code from developers? Otherwise they are not in violation of license.

Or is there place to report merely for being copycat?

**Comment by FireMasterK** [comment]
> I guess that the first required step would be to (1) install app (2) request app source code from developers? Otherwise they are not in violation of license as I understand.

It's actually illegal to publish binaries of a GPL 3.0 application (as done here in the play store), without also distributing the source code of the application. If the application owner said "Here's the source code of my probably ad-infested clone of LibreTube", or otherwise distributed the source code through some other means, I don't think we would have any problem since that would be legal.

> Or is there place to report merely for being copycat?

See https://github.com/TeamNewPipe/NewPipe/issues/539#issuecomment-1399222536 (mentioned in an earlier comment too)

> Making money from GPLv3 licensed code is not violating such license, and author being against it does not change it. There are some licenses that forbid commercial use or have "do no evil" clauses but they have own serious issues.

You're right, the real problem here is that they don't distribute the (modified) application's source code, which is against the GPL license as they distribute binaries.

I must also mention, LibreTube (and thereby any clone also) violates Google Play's Terms of Service, which is also why you don't find it on the play store, but only on F-Droid.

**Comment by terminal-cs** [comment]
This is why we can't have nice things. People need to stop taking for their own gain

**Comment by eclairevoyant** [comment]
> It's actually illegal to publish binaries of a GPL 3.0 application (as done here in the play store), without also distributing the source code of the application. 

It's not "illegal", it's a license violation which may or may not have legal implications depending on the jurisdiction.

> I must also mention, LibreTube (and thereby any clone also) violates Google Play's Terms of Service

Which part is a TOS violation? That might help get more attention/action on the app from Google, on top of the GPL violation and copycat nature of the app.

**Comment by yoguut** [comment]
This is most likely a policy violation, see here: https://support.google.com/legal/answer/3110420

I guess I would suggest to take the least effort approach by reporting the incident here: https://support.google.com/legal/answer/3110420

**Comment by Darin755** [comment]
I would contact the fsf's GPL enforcement team. If that doesn't work you could also try contacting the free software conservancy

**Comment by Bnyro** [comment]
Tutanota
